src/contracts/definitions/snip-721-def.ts

```diff
@@ -82,12 +91,14 @@ export const snip721Def: ContractDefinition = {
     },
 
     getApprovedForAll(
-      { address: owner }: Context,
+      { address: owner, key }: Context,
       include_expired?: boolean
     ): ContractQueryRequest {
+      let viewing_key = key ? key : null;
```

Comment:
Do we need to explicitly pass `null`? Can't we just simply rename `key` to `viewing_key`?

Reply:
yeah, we need to be explicit, if we pass `undefined` or `""` the contract thinks is a vk and tries to validate it (obviously that fails)

Reply:
Understood!